app/src/main/java/com/amaze/filemanager/ui/views/appbar/SearchView.java

```diff
@@ -52,20 +52,16 @@
 import android.content.Context;
```

Comment:
why are there this many changes to this file? pls change only the parts that are necessary. if you formatted the file with android studio's default formatter by mistake, revert it & run spoless. this huge of a change will break git history, and will be cumbersome to handle in the future.